Gemfile.lock

```diff
@@ -782,8 +782,8 @@ DEPENDENCIES
   representative_view
   roadie-rails (~> 1.0.3)
   roo (~> 2.7.0)
-  rspec-rails (>= 3.5.2)
   roo-xls (~> 1.1.0)
+  rspec-rails (>= 3.5.2)
```

Comment:
I just checked out master and bundled. I am on `bundler 16.1` and it wants gems in alphabetical order. This change was committed back in 2017, but recently merged last month. Maybe bundler was updated since then to fix an ordering issue.